### ISSUE TITLE
Unable to Copy Messages from Public Groups (empty: true)

### ISSUE THREAD

**Issue by X-Uchiha** [TelegramPlayGround/pyrogram#108]
### Checklist

- [X] I am sure the error is coming from Pyrogram's code and not elsewhere
- [X] I have searched in the issue tracker for similar bug reports, including closed ones
- [X] I ran `pip3 install -U https://github.com/pyrogram/pyrogram/archive/master.zip` and reproduced the issue using the latest development version

### Description

Attempting to copy messages from public groups returns "empty": true, making them inaccessible. The same code works correctly with messages from public channels.

Expected Behavior
Messages from public groups should be accessible and copyable, similar to channel messages.

### Steps to reproduce

1. Use copy_message or get_messages on any message from a public group.
2. Inspect the returned message payload, which shows "empty": true.

**Comment by SpEcHiDe** [comment]
>> Expected Behavior
Messages from public groups should be accessible and copyable, similar to channel messages.

Why / How can you **claim** this?

**Comment by X-Uchiha** [comment]
Sorry about the confusion—I thought group and channel messages were the same, so I made the report based on that. After testing with another library and the Telegram HTTP Bot API, I realized that group messages can't be copied unless the bot has access to them. 

Sorry for asking something I could test myself, but I only have one Telegram account! With a user client, can we copy protected messages directly like we can with a bot, or do we need to download them first?
